build/make-distribution.sh

```diff
@@ -381,6 +382,7 @@ else
     build_flink_client -Pflink-1.18
     build_flink_client -Pflink-1.19
     build_flink_client -Pflink-1.20
+    build_flink_client -Pflink-2.0
```

Comment:
which version of JDK do you expect the release manager to use?

Reply:
@pan3793, right. I have add the script to build java home for `flink-2.0,jdk-11` profile. PTAL.